vizro-core/docs/pages/user_guides/custom_tables.md

```diff
@@ -1,13 +1,13 @@
-# How to create custom tables
+# How to create custom Tables
 
-If you want to use the [`Table`][vizro.models.Table] model to and to create a custom [table](table.md) you can create your own custom table, e.g. when requiring computations that can be controlled by parameters.
+If you want to use the [`Table`][vizro.models.Table] model to create a custom [table](table.md), e.g. when requiring computations that can be controlled by parameters, then the below approach is recommended.
```

Comment:
Would remove the example - I think it's too specific and I didn't understand immediately what kind of table that is 😄 

```suggestion
If you want to use the [`Table`][vizro.models.Table] model to create a custom [table](table.md) then the below approach is recommended.
```